_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -47549,6 +47555,7 @@
 	dir = 1;
 	layer = 2.9
 	},
+/obj/item/melee/chainofcommand,
```

Comment:
Of course it had to be Purpose who added that.

Reply:
🍆